app/services/participant_validation_service.rb

```diff
@@ -44,8 +49,15 @@ def matching_record(trn:, nino:, full_name:, dob:)
     matches = 0
     trn_matches = padded_trn == dqt_record[:teacher_reference_number]
     matches += 1 if trn_matches
-    name_matches = full_name == dqt_record[:full_name]
+
+    name_matches = if check_first_name_only?
+                     full_name.split(" ").first.to_s == dqt_record[:full_name].split(" ").first.to_s
```

Comment:
I wonder, do we need `to_s` here? I would hope `split` would already give us a string.

I guess that makes sure neither value is `nil`, just not sure if it's worth it.

Reply:
hmm i think i glossed over this so it looks more like a mistake. i will remove the `to_s`. If one or both values are `nil` it shouldn't affect `==`.

Reply:
FYI I've now removed the `to_s` in a new commit